transformer_engine/pytorch/attention.py

```diff
@@ -95,6 +95,9 @@
 META_DO = tex.FP8BwdTensors.GRAD_INPUT2
 META_S = tex.FP8FwdTensors.GEMM3_OUTPUT
 META_DP = tex.FP8BwdTensors.GRAD_INPUT3
+# repurpose some unused amax history buffers for partial results of CP fwd and bwd
+META_O_CP = tex.FP8FwdTensors.GEMM2_WEIGHT
```

Comment:
@ksivaman , would it be ok to use the WEIGHT tensor here? Is there a possibility that `META_O_CP` gets updated/overwritten in `reduce_and_update_fp8_tensors()` when we don't really want to? The global buffer update code has changed and I'm wondering what controls the weight tensor update now and if it would affect `META_O_CP`.

Reply:
@xrennvidia The weight amaxes and non-weight amaxes have different schedules in the global buffer reduction. It's better that you stick to the INPUT/OUTPUT keywords here since `META_O_CP` and `META_DQKV_CP` are both activations. I suggest we use `META_O_CP = tex.FP8FwdTensors.GEMM2_OUTPUT`.

Reply:
Sounds good. Fixed.